### ISSUE TITLE
error while reading multiple empty rows separated by data  

### ISSUE THREAD

**Issue by gaurav-bodhare** [osglworks/java-excel-reader#7]

**Comment by greenlaw110** [comment]
@gaurav-bodhare can you provide a sample excel sheet and the source code that reproduce the error? Please also document the error details. thanks.

**Comment by gaurav-bodhare** [comment]
Here is the bean,I am using to map

`public class UserBean implements Serializable {

    private String employeeId;

    private String firstName;

    private String lastName;

    private String email;

    public String getEmployeeId() {
        return employeeId;
    }

    public void setEmployeeId(String employeeId) {
        this.employeeId = employeeId;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}`

and here is the method I am using to read excel file
`public List<UserBean> readExcelOfUsers(File file) {
        ExcelReader reader = ExcelReader.builder()
            .map("Employee Id").to("employeeId")
            .map("First Name").to("firstName")
            .map("Last Name").to("lastName")
            .map("Email Id").to("email")
            .file(file)
            .build();
        List<UserBean> beans = reader.read(UserBean.class);
        return beans;
    }`

I am using maven and added following dependency for this excel-reader
`<dependency>
            <groupId>org.osgl</groupId>
            <artifactId>excel-reader</artifactId>
            <version>1.0.0</version>
 </dependency>`

The file which I was trying to read 
https://drive.google.com/file/d/10FaA28IkWRm2EdoHu0PT9UsCUxViC2pI/view?usp=sharing

**Comment by greenlaw110** [comment]
The error stack:

```
Exception in thread "main" java.lang.NullPointerException
	at org.osgl.xls.ExcelReader$TolerantLevel.onReadCellException(ExcelReader.java:302)
	at org.osgl.xls.ExcelReader.read(ExcelReader.java:166)
	at org.osgl.xls.ExcelReader.read(ExcelReader.java:122)
	at org.osgl.xls.ExcelReader.read(ExcelReader.java:82)
	at demo.excel.UserBean.readExcelOfUsers(UserBean.java:59)
	at demo.excel.UserBean.main(UserBean.java:64)
```

**Comment by greenlaw110** [comment]
@gaurav-bodhare fixed in 1.4.1-SNAPSHOT. can you give it a try?
